src/ring_theory/multiplicity.lean

```diff
@@ -57,6 +57,18 @@ begin
     apply _root_.le_antisymm; { apply nat.find_mono, norm_cast, simp } }
 end
 
+example (a b : ℤ) : a ∣ b ↔ a ∣ b.nat_abs := int.dvd_nat_abs.symm
```

Comment:
Do you want to leave this in the code? Or were you planning to remove it?

Reply:
Oops, my bad